### ISSUE TITLE
Duplicate docket entries (1 from rss?)

### ISSUE THREAD

**Issue by johnhawkinson** [freelawproject/recap#272]
In this case, https://www.courtlistener.com/docket/6296809/calderon-jimenez-v-cronen/?page=2, it appears that RECAP has two entries for the same order. Presumably one from RSS and one from the docket report and they did not merge properly:

<img width="1171" alt="Mar 21, 2019;  Order on Motion for Extension of Time; <p>223; Mar 21, 2019; Judge Mark L. Wolf: ELECTRONIC ORDER entered granting 222 Motion for Extension of Time (Bono, Christine) (Entered: 03/21/2019); Main Doc­ument" src="https://user-images.githubusercontent.com/1270718/54861554-20140600-4d01-11e9-82ef-242c6335793e.png">

"Huh."

**Comment by mlissner** [comment]
Hm, somehow we got that as an unnumbered entry though it seems to be a numbered entry now. That's pretty weird.

**Comment by johnhawkinson** [comment]
interim conclusion: we need to save raw rss for debugging. Maybe for other things too.

**Comment by mlissner** [comment]
Yeah, wouldn't hurt. We can open a bug for it, but I have no idea when I'll get to it.

**Comment by johnhawkinson** [comment]
This is not a one-off. Today's example: https://www.courtlistener.com/docket/6151669/alasaad-v-duke/?filed_after=&filed_before=&entry_gte=&entry_lte=&order_by=desc

<img width="1018" alt="Screen Shot 2019-04-11 at 23 17 21" src="https://user-images.githubusercontent.com/1270718/56010057-0e929e00-5cb0-11e9-8845-8987b78719e4.png">

**Comment by johnhawkinson** [comment]
And then there's this peculiar variant, where it looks a "Notice of Consent - No Consent" was edited to "All parties have consented" but both appear in the RECAP docket report, one unnumbered. I wonder if this means if we had reparsed the RSS feed after the edit (e.g. w/i 24 hours), we would have gotten the proper information.

https://www.courtlistener.com/docket/6106337/dalessio-v-university-of-washington/?filed_after=&filed_before=&entry_gte=&entry_lte=&order_by=desc

<img width="957" alt="Screen Shot 2019-06-11 at 08 03 05" src="https://user-images.githubusercontent.com/1270718/59270424-68b3cf00-8c1f-11e9-970e-a716bcec8a32.png">


So the duplication is a bigger problem in the face of edits.

**Comment by ikeboy** [comment]
Still an issue, e.g. https://www.courtlistener.com/docket/16554647/infinity-global-consulting-group-inc-v-tilray-inc/

![image](https://user-images.githubusercontent.com/12200693/75069296-e2929c80-54be-11ea-9b39-60b47f5201b1.png)
![image](https://user-images.githubusercontent.com/12200693/75069309-e9b9aa80-54be-11ea-80dd-ea248f827916.png)

**Comment by johnhawkinson** [comment]
Yes, this is, unfortunately, a regular problem. It's pretty confusing, we should prioritize fixing it, and then somehow managing to repair the months of busted data it has lead to :(

**Comment by mlissner** [comment]
Is there a way to fix it? The only thing these items have in common, I think, is the date?

**Comment by johnhawkinson** [comment]
Oh, it's certainly fixable. This problem was introduced when changes were made with respect to `de_seqno` and ordering, or something like that.  I thought we had a conversation around the change that introduced this bug, perhaps in the Slack. In any event, it was shortly prior to my opening this Issue, so probably Feb or March 2019.
